examples/learn/async/custom_client/decorator/mistral/messages.py

```diff
@@ -1,7 +1,7 @@
 from mirascope.core import Messages, mistral
-from mistralai.async_client import MistralAsyncClient
+from mistralai import Mistral
 
 
-@mistral.call("mistral-large-latest", client=MistralAsyncClient())
+@mistral.call("mistral-large-latest", client=Mistral())
```

Comment:
i believe in all of these example instances we need to manually load and pass in the api key from the environment? at least that was my understanding from their docs